workers/generic-worker/interactive/conpty.go

```diff
@@ -0,0 +1,289 @@
+//go:build windows
+
```

Comment:
Mind just updating the filename to `conpty_windows.go`? This'll also make it clear from the filetree within the repo this code is specific to windows just from the filename.

```suggestion
```